package.json

```diff
@@ -121,6 +122,12 @@
         "@dfinity/candid",
         "@dfinity/principal"
       ]
+    },
+    {
+      "name": "@dfinity/rosetta-client",
```

Comment:
fyi if the build fails you might need to provide the PR without these, then merge the PR and then provide these lines. It compares main vs branch therefore might need first to be merged

Reply:
Not sure sure

Reply:
I tried building with

```
npm run build --workspaces
```

And it works well. No errors.